docs/scalardb-supported-databases.md

```diff
@@ -1,31 +1,141 @@
-| ScalarDB 2.3 |  ✅  |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.2 |  ✅  |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.1 |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
-| ScalarDB 2.0 |  ✅  |  ✅  |  ✅  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  ❌  |  |
+# Supported Databases
+
+ScalarDB supports the following databases and their versions.
+
+## Amazon DynamoDB
+
+| Version | Amazon DynamoDB |
+|:------------------|:---------------|
+| **ScalarDB 3.10** | ✅ |
+| **ScalarDB 3.9** | ✅ |
+| **ScalarDB 3.8** | ✅ |
+| **ScalarDB 3.7** | ✅ |
+| **ScalarDB 3.6** | ✅ |
+| **ScalarDB 3.5** | ✅ |
+| **ScalarDB 3.4** | ✅ |
+
+## Apache Cassandra
+
+{% capture notice--info %}
+**Note**
+
+For requirements when using Cassandra or Cassandra-compatible databases, see [Cassandra or Cassandra-compatible database requirements](requirements/#cassandra-or-cassandra-compatible-database-requirements).
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+| Version | Apache Cassandra 4.1 | Apache Cassandra 4.0 | Apache Cassandra 3.11 | Apache Cassandra 3.0 | Apache Cassandra 2.2 |
+|:------------------|:---------------|:-----------------|:----------------|:-----------------|:----------------|
+| **ScalarDB 3.10** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.9** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.8** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.7** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.6** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.5** | ❌ | ❌ | ✅ | ✅ | ✅ |
+| **ScalarDB 3.4** | ❌ | ❌ | ✅ | ✅ | ✅ |
+
+## Azure Cosmos DB for NoSQL
+
+| Version | Azure Cosmos DB for NoSQL |
+|:------------------|:---------------|
+| **ScalarDB 3.10** | ✅ |
+| **ScalarDB 3.9** | ✅ |
+| **ScalarDB 3.8** | ✅ |
+| **ScalarDB 3.7** | ✅ |
+| **ScalarDB 3.6** | ✅ |
+| **ScalarDB 3.5** | ✅ |
+| **ScalarDB 3.4** | ✅ |
+
+## JDBC databases
+
+{% capture notice--info %}
+**Note**
+
+For recommendations when using JDBC databases, see [JDBC database recommendations](requirements/#jdbc-database-requirements).
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
+
+### Amazon Aurora MySQL
+
+| Version | Amazon Aurora MySQL 3 | Amazon Aurora MySQL 2 |
+:---------------|:---------------|:---------------|
+| **ScalarDB 3.10** | ✅ | ✅ |
+| **ScalarDB 3.9** | ✅ | ✅ |
+| **ScalarDB 3.8** | ✅ | ✅ |
+| **ScalarDB 3.7** | ✅ | ✅ |
+| **ScalarDB 3.6** | ✅ | ✅ |
+| **ScalarDB 3.5** | ✅ | ✅ |
+| **ScalarDB 3.4** | ✅ | ✅ |
+
+### Amazon Aurora PostgreSQL
+
+| Version | Amazon Aurora PostgreSQL 14 | Amazon Aurora PostgreSQL 13 | Amazon Aurora PostgreSQL 12 |
```

Comment:
@brfrn169 Based on the comment in https://github.com/scalar-labs/scalardb/pull/1012#discussion_r1302373516, I think we should add Amazon Aurora PostreSQL 15 to this table. 

Do ScalarDB versions 3.4 to 3.10 all support Amazon Aurora PostreSQL 15?

Reply:
**Note:** See https://github.com/scalar-labs/scalardb/pull/1020#discussion_r1303972487